fdbserver/ClusterController.actor.cpp

```diff
@@ -3959,7 +3951,15 @@ ACTOR Future<Void> timeKeeper(ClusterControllerData* self) {
 				Version v = tr->getReadVersion().get();
 				int64_t currentTime = (int64_t)now();
 				versionMap.set(tr, currentTime, v);
+				if (!g_network->isSimulated()) {
+					// This is done to provide an arbitrary logged transaction every ~10s.
+					// FIXME: replace or augment this with logging on the proxy which tracks
+					//       how long it is taking to hear responses from each other component.
 
+					UID debugID = deterministicRandom()->randomUniqueID();
+					TraceEvent("TimeKeeperCommit", debugID).detail("Version", v).detail("Timestamp", currentTime).log();
```

Comment:
It looks like `currentTime` is just an integer version of the `Time` field that already exists. Can the `Time` field serve this purpose?

Reply:
I think for this event we want to log timestamp we put in the map. Later we could figure out the timestamp-version mapping using the logs.

Reply:
Yeah, what I mean is that we are logging the timestamp already in the `Time` field that's included in every trace event. It's just being logged as a double instead of an int, but otherwise this new field would be a duplicate of the existing data.